2_segmenting_data/segmentation_utils.py

```diff
@@ -0,0 +1,228 @@
+# from cellpose.io import logger_setup
+from cellpose import models, core, io, utils
+from typing import Tuple
+
+import pathlib
+import pandas as pd
+import skimage.io as io
+
+import cv2
+import numpy as np
+
+import matplotlib.path as mplPath
+
+
+def get_seg_data(image: np.ndarray, model_specs: dict, data: str) -> pd.DataFrame:
+    """finds center X,Y or outlines of objects using cellpose specs from model_specs and return pandas array with the specified data of objects
+    Args:
+        image (np.ndarray): image with objects to segment
+        model_specs (dict): specifications for cellpose segmentation
+        data (str): specified segmentation data information for an object, either "locations" or "outlines"
```

Comment:
Can you give this variable a slightly more descriptive name?

maybe even "outlines_or_locations" would be better?

Reply:
or `seg_data_output`?

Reply:
I like that last one!